server/statistics/metrics.go

```diff
@@ -30,7 +30,7 @@ var (
 			Subsystem: "scheduler",
 			Name:      "store_status",
 			Help:      "Store status for schedule",
-		}, []string{"namespace", "address", "store", "type"})
+		}, []string{"address", "store", "type"})
```

Comment:
Will it cause compatibility problem when upgrade from old version?

Reply:
Yes, it will. The monitor should also be upgraded under this situation.

Reply:
Will old data be lost?

Reply:
Yes. It will. Any better idea?

Reply:
seems no better idea. :(